.eslintrc

```diff
@@ -1,7 +1,6 @@
 {
 	"extends": "airbnb",
 	"parser": "babel-eslint",
-	"ignorePatterns": ["**/*.stories.js"],
```

Comment:
I think it is useful to avoid all lint errors from stories.

It seems we cannot use ignorePatterns when extending another config, see https://eslint.org/docs/user-guide/configuring#how-it-works

Should try .eslintignore maybe ?